src/routes/userapp/components/main-page-specific/ItemListElement.tsx

```diff
@@ -62,30 +62,35 @@ function ItemListElement({ item, config }: ItemListElementProps) {
       >
         <Box flexGrow={1} marginRight={2}>
           <Typography variant="h5">{item.attributes.title}</Typography>
-          <Typography>{item.attributes.subtitle}</Typography>
-
-          {config.mainPage && config.mainPage.showRating && item.mark && (
-            <Ratings mark={item.mark} />
+          {item.attributes.subtitle && (
```

Comment:
A tu nie trzeba jeszcze sprwadzić w storeConfigu czy w ogóle należy to wyświetlac?

Reply:
export type MainPage = {
  welcomeTextLine1: string;
  welcomeTextLine2: string;
  enableFiltering: boolean;
  showItemTitle: boolean;
  showItemSubtitle: boolean;
  showItemImg: boolean;
  showRating: boolean;
};

export type DetailsPage = {
  showRating: boolean;
  showComments: boolean;
  showItemDescription: boolean;
  showSubItemTitle: boolean;
  showSubItemSubtitle: boolean;
  reservationConfirmationPrompt: string;
  reservationFailurePrompt: string;
  reservationSummaryPrompt: string;
};

zarówno subtitle i image nie są tutaj definiwoane, wiec defaultowo tam sie wyswietla jesli jest

Reply:
![image](https://github.com/AleksandraWolska/ZPIFrontend/assets/69108816/dd1cc546-54c9-4fee-a15c-20ba0299c098)

Na pewno? W sensie to szczegół tylko, ale chyba jednak są

Reply:
to pierwsze do mainpage, i tam zmodyfikowalam właśnie,
A w detailsach jest SubItemTitle, nie ItemSubTitle